### PR TITLE
Install latest stable nginx during bootstrap

### PR DESCRIPTION
**Pull request by enykeev**
The change is required by https://github.com/StackStorm/st2/pull/2560 since the version from official ubuntu repos is too old and doesn't allow you to change headers on 5xx responses.

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
One could argue that we will mess with customers' existing nginx installation but we anyways require a completely new box to do this. LGTM.

File: scripts/st2bootstrap-deb.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-USERNAME='test'
+USERNAME='st2admin'
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: weird.

**Comment by Kami** [comment]
Yeah, for "appliance" single box style deployments (which is what those scripts do), we should control all the software versions (nginx, mongodb, rabbitmq).

For everything else (if people want to install packages manually), we should document minmum required / supported versions of software we need.

**Comment by lakshmi-kannan** [comment]
+1 to merge this to v1.3

File: scripts/st2bootstrap-deb.sh (hunk 1 above)
Comment (enykeev) [inline scripts/st2bootstrap-deb.sh]: indeed.

**Pull request merged**